README.md

```diff
@@ -424,7 +426,48 @@ Assumes database resolution is done by adding prefixes. If database resolution i
      where h.date = '2016-05-13'
        and h.hour = 1
     ;
+
+## Database Name Mapping
+Waggle Dance allows configuration to map database names. This feature is intended to be useful when migrating from an existing database into different environments.
+A use case:
+We have legacy data lakes that have tables belonging to the same domain in differently named databases. E.g. a booking domain is called 'datawarehouse' or 'hotel_booking'. When federating to those datalakes it helps data discovery if those names are standardized to one name. Renaming a database is easy but migrating ETL might not be, so to make migration easier you can configure Waggle Dance to remap the names.
+
+Example Datalake X federating to 2 other "legacy" datalakes Y and Z.
+This is what we want the different databases to look like in the datalakes:
+
+X has:
```

Comment:
X doesn't actually have these right, it's what we want the names to look like from Waggle Dance? X might have its own booking database I guess? Not sure if we should put that in the example, might make it more confusing?

Reply:
yes it doesn't and yes I tried to keep it to a minimum, it's tricky :)